addons/website_payment/static/src/js/website_payment_donation.js

```diff
@@ -35,4 +37,31 @@ publicWidget.registry.WebsitePaymentDonation = publicWidget.Widget.extend({
             donationCommentEl.value = "";
         }
     },
+    /**
+     * @private
+     */
+    _onChangeAmountInput() {
+        const inputEl = document.querySelector("#other_amount_value");
+        const warningMessageEl = document.querySelector("#warningMessageId");
+        const warningMinMessageEl = document.querySelector("#warningMinMessageId");
+        const value = parseFloat(inputEl.value);
+        warningMessageEl.classList.toggle("d-none", Boolean(value));
+        if (value) {
+            warningMinMessageEl.classList.toggle("d-none", !(inputEl.min && inputEl.min >= value));
```

Comment:
can you please explain this `!(inputEl.min && inputEl.min >= value)` ?

Reply:
If there are no prefilled options available, the form will redirect to the standard input form, which uses the <t t-call="website_payment.donation_input"/> template. Consequently, there will be no minimum value set for the donation input field, leading to a display without any constraints, as illustrated in the image below.

![image](https://github.com/user-attachments/assets/26a9381f-a6f2-494b-b111-0f2d80427745)